decoder.tex

```diff
@@ -258,14 +259,22 @@ \section{Decoder pseudo code}
 
   return FALSE
 
-# Determine if instruction is an uninferable discontinuity #
-function is_uninferable_discon (instr)
+# Determine if instruction is a return from trap #
+function is_return_from_trap (instr)
 
-  if (is_uninferable_jump(instr) or
-      (instr.opcode == URET)      or
+  if ((instr.opcode == URET)      or

```

Comment:
may be a good opportunity to drop URET which was never ratified.

Reply:
Yes, happy to take this out.